axe-check.js

```diff
@@ -46,10 +46,9 @@ async function runAccessibilityCheck() {
       report += nodes;
       report += '</details>\n\n';
     }
-
   }
   fs.writeFileSync('accessibility_report.md', report, 'utf8');
   console.log('Accessibility report generated.');
 }
 
-runAccessibilityCheck();
+runAccessibilityCheck();
```

Comment:
You seem to be having an issue in your axe-check.js, I think is a no-next-line error or something, try giving it a new line after that line 54